templates/quickstart-hashicorp-vault-master.template

```diff
@@ -289,50 +288,50 @@ Parameters:
     Description: The HashiCorp Vault name for the AWS IAM Role.
   VaultVersion:
     Type: String
-    Description: Which version of HashiCorp Vault to install
+    Description: Specify which version of HashiCorp Vault to install.
     Default: 1.4.0
     AllowedValues:
     - 1.4.0
   LoadBalancerType:
     Type: String
     Default: "Internal"
-    Description: Is the Load Balancer for Vault internal or external.
+    Description: Specify if the load balancer for HashiCorp Vault is internal or external.
     AllowedValues:
     - "Internal"
     - "External"
   DomainName:
     Type: String
-    Description: The fully qualified domain name for the HashiCorp Vault load balancer. Use with 'HostedZoneID' if you are not providing 'ACMSSLCertificateArn'.
+    Description: Fully qualified domain name for the HashiCorp Vault load balancer. If you don't provide a value for ACMSSLCertificateArn, use the HostedZoneID.
     MaxLength: 128
     Default: ""
   HostedZoneID:
     Type: String
-    Description: Route 53 Hosted Zone ID of the domain name. Used in conjunction with 'DomainName' if you are not providing 'ACMSSLCertificateArn'.
+    Description: Route 53-hosted zone ID of the domain name. If you don't provide an ACMSSLCertificateArn value, the Quick Start creates the ACM certificate for you using HostedZoneID in conjunction with DomainName.
     Default: ""
   ACMSSLCertificateArn:
-    Description: The Amazon Resource Name (ARN) of the SSL certificate to use for the load balancer. Use 'ACMSSLCertificateArn' if you are not using 'DomainName' and 'HostedZoneID'.
+    Description: Amazon Resource Name (ARN) of the load balancer's SSL certificate. If you don't provide values for DomainName and HostedZoneID, provide a value for ACMSSLCertificateArn.
     Type: String
     Default: ""
   VaultKubernetesEnable:
-    Description: Enable Kubernetes authentication and create Kubernetes authentication role.
+    Description: Enables Kubernetes authentication and creates a Kubernetes authentication role.
     Type: String
     Default: 'false'
     AllowedValues:
       - 'true'
       - 'false'
   VaultKubernetesRoleName:
-    Description: Internal HashiCorp Vault name for Kubernetes authentication role.
+    Description: Internal Vault name for the Kubernetes authentication role.
     Type: String
     Default: 'kube-auth-role'
   VaultKubernetesHostURL:
-    Description: "URL of Kubernetes cluster eg: https://192.168.99.100:8443."
+    Description: "URL of Kubernetes cluster (e.g., https://192.168.99.100:8443)."
     Type: String
   VaultKubernetesCertificate:
-    Description: "AWS SSM Parameter containing base64 encoded PEM CA certificate of Kubernetes cluster service account."
+    Description: "AWS SSM Parameter containing a base64-encoded PEM CA certificate for the Kubernetes cluster service account."
```

Comment:
"for the" seems wrong here
This is an existing property of the customers existing Kubernetes cluster that they are providing to configure the Vault cluster with.

Reply:
Would it be correct if we said "... used for the Kubernetes ..."?

"... of the Kubernetes ..." also works, assuming that the certificate belongs to (i.e., is owned by) the "Kubernetes cluster service account."

Reply:
Yes the certificate does belong the the cluster service account. Used for isn't right because it exists and would be used for configuring Vault to allow the Kubernetes service account to integrate. I think the " of the " option will be clear to the audience.